metadata-ingestion/tests/integration/feast/wait-for-it.sh

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
+# Use this script to test if a given TCP host/port are available
+# from https://github.com/vishnubob/wait-for-it
```

Comment:
dont we already have this file somewhere in the repo?

Reply:
Not sure... @hsheth2 do you know of any we have lying around?

Reply:
I couldn't find any shell scripts that do the same, but I think what you had in mind was the `wait_for_port` function that we have in `tests.test_helpers.docker_helpers`. In this case we need this shell script for docker-compose.

Reply:
We used to have it in the repo but I removed them when I did the docker-compose test harness overhaul. In this case we do actually need the script so adding it here is fine